src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html

```diff
@@ -42,25 +56,32 @@
           <div class="input-group">
             <select class="form-control" ngbTooltip="The time zone for the course. You should not need to change this as it is auto-detected based on your
               device settings. TEAMMATES automatically adjusts to match the current time offset in your area, including clock changes due to daylight saving time."
-              [(ngModel)]="course.timeZone" [disabled]="!isEditingCourse">
+              [(ngModel)]="course.timeZone" #timeZone="ngModel" [disabled]="!isEditingCourse" required name="timeZone"
+              [ngClass]="{'invalid' : timeZone.invalid && (timeZone.touched || timeZone.dirty),
+              'valid' : timeZone.valid && (timeZone.touched || timeZone.dirty)}">
```

Comment:
Hmm I'm not sure if you need these validations for a dropdown list? In this case, the timezone can take any value and it won't be empty. (The default is UTC.)

Reply:
Yea you're right, timezone validation doesn't seem necessary